repo/meta/index.json

```diff
@@ -244,6 +244,20 @@
       }
     },
     {
+      "currentVersion":"1.1.0-RC1",
+      "description":"A cluster-wide init and control system for services in cgroups or Docker containers.",
+      "framework":true,
+      "name":"marathon-rc",
```

Comment:
Since multiverse was dropped, we now have `marathon` and `marathon-rc` as different packages.